app/controllers/jobs/new.js

```diff
@@ -76,6 +78,8 @@ export default class JobsNewController extends Controller {
 
   @action
   async scheduleHarvestJob() {
+    this.scheduling = true;
```

Comment:
It might be worth converting these to ember-concurrency tasks so that you can use the derived state instead. You can then also use a dropTask so it automatically drops new tasks if one is already running.